src/test/groovy/com/netflix/frigga/NamesSpec.groovy

```diff
@@ -341,7 +341,7 @@ class NamesSpec extends Specification {
         'cass-nccpintegration-random-junk-c0northamerica-d0prod-h0gamesystems-p0vizio-r027-u0nccp-w0A-z0useast1a' == names.cluster
         'cass' == names.app
         'nccpintegration' == names.stack
-        'random-junk' == names.detail
+        'random-junk-c0northamerica-d0prod-h0gamesystems-p0vizio-r027-u0nccp-w0A-z0useast1a' == names.detail
```

Comment:
changes in this test highlight the breaking change behavior from refactoring out the labeled variables into a NamingConvention